web/themes/custom/recipe_theme/src/sass/view/_view.scss

```diff
@@ -0,0 +1,12 @@
+#edit-actions {
+  margin-bottom: 50px;
+}
+
+#edit-search-api-fulltext {
```

Comment:
As I mentioned before, don't use float unless it's for images and text. Please try to [use flex display](https://css-tricks.com/snippets/css/a-guide-to-flexbox/), probably not on this element but a wrapper one.

Also, unless necessary please add theming for classes instead of ids.

Whenever creating styles also think about the mobile users, it's also a good practice to start developing for mobile screens and then add rules for larger screens with `min-width` media queries. Here's a very good example how to use SASS mixins with media queries: https://www.freecodecamp.org/news/the-100-correct-way-to-do-css-breakpoints-88d6a5ba1862/#tip-3-be-declarative